website/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '2'
+
+services:
+  nginx-proxy:
+    image: jwilder/nginx-proxy
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - "/etc/nginx/vhost.d"
+      - "/usr/share/nginx/html"
+      - "/var/run/docker.sock:/tmp/docker.sock:ro"
+      - "/etc/nginx/certs"
+    environment:
+      ENABLE_IPV6: "false" # Enable when ready
+
+  letsencrypt-nginx-proxy-companion:
+    image: jrcs/letsencrypt-nginx-proxy-companion
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+    volumes_from:
+      - "nginx-proxy"
+    environment: 
+      ACME_CA_URI: https://acme-staging.api.letsencrypt.org/directory # testing value
+
+  jekyll:
```

Comment:
I guess we need to put `github.com/tomeshnet/tomesh.net` as source and add the webhook?

Reply:
something like that!